pychunkedgraph/app/common.py

```diff
@@ -13,13 +14,23 @@
 
 USER_NOT_FOUND = "-1"
 
+try:
+    # if key set assume True
+    _ = os.environ["PCG_SERVER_LOGS_LEAVES_MANY"]
+    LOG_LEAVES_MANY = True
+except KeyError:
+    LOG_LEAVES_MANY = False
+
```

Comment:
why not `LOG_LEAVES_MANY = os.environ.get("PCG_SERVER_LOGS_LEAVES_MANY", False)` ?

Reply:
that works too but usually not a good idea to mix types, `os.environ[key]` always returns `str`